src/Agent.Worker/AgentPluginManager.cs

```diff
@@ -160,18 +160,35 @@ public async Task RunPluginTaskAsync(IExecutionContext context, string plugin, D
 
             // construct plugin context
             var target = context.StepTarget();
+            Variables.TranslationMethod translateToHostPath = null;
```

Comment:
Even though I think this should work fine, setting a method to null like this makes me a little uncomfortable - any chance we can just make DefaultStringTranslator public and just set it to that here?

Reply:
I think it specifically makes me uncomfortable because we go from it being initialized with a value to being set to null which basically just means still the default value.

Reply:
ok, let me see if I can improve this a big